main.py

```diff
@@ -1,18 +1,25 @@
 import yaml
+import logging
 import os
+import logging.config
+import yaml
 
 from flask import Flask
 from apis import api
+from colored_logging import ColorFormatter
```

Comment:
Instead of making a new file. Can we put the color formatter in utils.py and add other future handy functions from there. Or, as this is matter of configuration how about in config.py 

Your call

Reply:
Moved things around but i would still prefer to have a separate file for it as it's logically not the part of the "apis"..